torch/csrc/Module.cpp

```diff
@@ -215,6 +215,21 @@ static PyObject* THPModule_crashIfATenASAN(PyObject* module, PyObject* arg) {
   return THPUtils_packInt32(at::_crash_if_asan(THPUtils_unpackInt(arg)));
 }
 
+static PyObject* THPModule_crashIfDebugAssertsFail(
+    PyObject* module,
+    PyObject* arg) {
+  THPUtils_assert(
+      THPUtils_checkLong(arg),
+      "crash_if_debug_asserts_fail expects an int set to a nonzero value during debug builds, "
+      "but got %s",
```

Comment:
This is a data type check, so it should say something like
```suggestion
      "crash_if_debug_asserts_fail expects an integer argument,  but got %s",
```